lldb/test/Shell/Commands/command-breakpoint-by-addr.test

```diff
@@ -0,0 +1,12 @@
+# RUN: %clang_host -g -O0 %S/Inputs/main.c -o %t.out
+# RUN: %lldb -b -o 'breakpoint set -a "main+26"' \
```

Comment:
Nested quotes don't work (or maybe work differently) on windows. Better put this into a command file (i.e., this file, since it's just full of comments ).